nox.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nox
+
+
+@nox.session
+def lint(session):
+    session.interpreter = 'python3'  # So it understands Python3 syntax
```

Comment:
Specify the version like `python3.6`?

Reply:
Any version 3.0 or above works.  I dug deeper, and the real problem is a Unicode problem with Python 2.  So I updated the comment.  Note that regular Python installs, and `virtualenv -p python 3.anything` creates a `bin/python3` symlink, so this should ok.